package.json

```
@@ -213,7 +213,10 @@
     "nonce-tracker@npm:^3.0.0": "patch:nonce-tracker@npm%3A3.0.0#~/.yarn/patches/nonce-tracker-npm-3.0.0-c5e9a93f9d.patch",
     "@trezor/schema-utils@npm:1.0.1": "patch:@trezor/schema-utils@npm%3A1.0.1#~/.yarn/patches/@trezor-schema-utils-npm-1.0.1-70c2a68232.patch",
     "lavamoat-core@npm:^15.1.1": "patch:lavamoat-core@npm%3A15.1.1#~/.yarn/patches/lavamoat-core-npm-15.1.1-51fbe39988.patch",
-    "@metamask/snaps-sdk": "^4.0.1",
+    "@metamask/snaps-sdk": "4.0.1",
```

```suggestion
    "@metamask/snaps-sdk": "4.0.1 || ^4.2.1",
```
(could) Just avoid bad versions with something like this
(which are the bad ranges btw?)

Also e.g. `npm deprecate @metamask/snaps-sdk@4.1.0` to make it not being pulled in by package managers anywhere, in case this affects more than just metamask-extensuion.

---

There isn't really a bad version, its just the RC doesn't have all of the required work extension side to deal with the breaking changes: https://github.com/MetaMask/metamask-extension/commit/3a05ae029eadf97ea7488b01e9e331fd94dacc1e

Instead of complicating the RC further, it seemed easiest to stick to the intended versions of the packages